CDTDatastore/CDTReplicator/CDTReplicator.m

```diff
@@ -50,7 +50,33 @@ @interface CDTReplicator ()
 @property (nonatomic) BOOL started;
 
 @end
-
+/*
+    The replicatore supports "Fire and forget" this is where the application releases it's reference
+    to the replicator after it has called -startWithGroup:Error (or methods that call into it). For
+    this mode to be supported, the instance of CDTReplicator must retain itself so it doesn't become
+    dealloacated during replication.
```

Comment:
Say how: "Each instance retains itself by assigning the instance to its own XXX property."